### DISCUSSION TITLE
chia钱包的同步文件能否不要 放在C盘里面，我电脑的C盘空间满来，最好能放在D盘里面，有人直到怎么做吗？

### DISCUSSION THREAD

**Discussion by lu6zhi6gan** [Chia-Network/chia-blockchain#7632] [Support]
chia钱包的同步文件能否不要 放在C盘里面，我电脑的C盘空间满来，最好能放在D盘里面，有人直到怎么做吗？

**Comment by keliew** [comment]
https://github.com/Chia-Network/chia-blockchain/discussions/7543

**Comment by lu6zhi6gan** [comment]
有没有中文版的。。。英语水平有限，网页翻译不是很灵光

**Reply by keliew** [reply]
> 有没有中文版的。。。英语水平有限，网页翻译不是很灵光

then use baidu...
这种操作教程多的是  跟奇亚没多大的关系  百度便知
